core-web/libs/portlets/edit-ema/portlet/src/lib/store/features/editor/withEditor.ts

```diff
@@ -55,8 +55,10 @@ const buildIframeURL = ({ url, params, isTraditionalPage }) => {
         return new String('');
     }
 
+    // Remove trailing slash from host
+    const host = (params.clientHost || window.location.origin).replace(/\/$/, '');
     const pageURL = getFullPageURL({ url, params, userFriendlyParams: true });
-    const iframeURL = new URL(pageURL, params.clientHost || window.location.origin); // Add Host to iframeURL
+    const iframeURL = new URL(`${host}/${pageURL}`);
```

Comment:
you can add the host as second argument here

Reply:
That's the issue; if you pass the host as a second argument and it includes a page path, such as http://localhost:8080/angular-app, only http://localhost:8080 will be considered, and the /angular-app portion will be ignored. 

https://developer.mozilla.org/es/docs/Web/API/URL/URL

Reply:
Ohh, nice catch